transform/app/transform/utils/delete.py

```diff
@@ -0,0 +1,26 @@
+import os
+import logging
+import shutil
+
+from app.settings import settings
+
+logger = logging.getLogger(__name__)
+
+
+def clean_tmp():
+    """
+    Deletes all files in the _tmp directory.
```

Comment:
Dont hardcode _tmp in the doc :D 
Deletes all files from the TMP_DIRECTORY

Reply:
My mistake, I forgot to put it here.